aws/opsworks_layers.go

```diff
@@ -292,12 +297,15 @@ func (lt *opsworksLayerType) Read(d *schema.ResourceData, client *opsworks.OpsWo
 		d.Set("short_name", layer.Shortname)
 	}
 
-	if v := layer.CustomJson; v == nil {
-		if err := d.Set("custom_json", ""); err != nil {
-			return err
+	v := layer.CustomJson
+	if v == nil {
+		d.Set("custom_json", "")
+	} else {
+		policy, err := structure.NormalizeJsonString(*v)
```

Comment:
Nitpick: The short variable name `v` was ok-ish when it was in the previously short context, but I think we should now just reference `layer.CustomJson` in all 3 places for clarity.

Reply:
Sure thing